build-scripts/post_build.sh

```diff
@@ -23,6 +23,22 @@ download_and_extract() {
   fi
 }
 
+build_biowardobe_ng() {
+  if [ -e $SATDIR/main.js ]; then
+    warn "biowardrobe-ng has been already built. Skipping compilation"
+  else
+    echo "Building biowardrobe-ng from $1"
+    cd $1
+    meteor npm install
```

Comment:
Better use `npm install` when we build our local version is going to be used. and meteor's version could be different. Like now we have 12.19 but meteor something different